apps/reader/src/components/Reader.tsx

```diff
@@ -299,7 +299,11 @@ function BookPane({ tab, onMouseDown }: BookPaneProps) {
         tab.showPrevLocation()
         return
       }
-      if (mobile === false && el.tagName === 'IMG') {
+      if (
+        mobile === false &&
+        el.tagName === 'IMG' &&
+        el.classList.contains('imgepub')
```

Comment:
what is "imgepub"? I think check whether `src` starts with `blob:` is better here.

Reply:
I thought it was the classname which epub.js added to images, but I was wrong, it seems from the special epub sample file I inspected.
changed to `blob:` now.